### ISSUE TITLE
true && exit Panic

### ISSUE THREAD

**Issue by aeosynth** [redox-os/ion#332]
true && exit
    ion: command not found: exit

**Comment by huntergoldstein** [comment]
It seems like the problem has to do with builtins being evaluated before a pipeline is constructed. You can replace `exit` with any builtin command such as `cd`.

**Comment by huntergoldstein** [comment]
My mistake, it turns out there's a [list of hard coded builtins](https://github.com/redox-os/ion/blob/9b936d6493a889784d34fc4509710d8ac20d9409/src/shell/job.rs#L79). I'll fix this issue by updating the list but there ought to be a more permanent fix (read: I'll open another issue).

**Comment by mmstick** [comment]
This is a matter of only getting the first job read and executed when a builtin is detected, which is easily fixed.

**Comment by mmstick** [comment]
@huntergoldstein Are you still looking into this? If not, I can work on it.

**Comment by huntergoldstein** [comment]
I'm working on #316 right now; only briefly looked into this.

**Comment by mmstick** [comment]
Merging this issue with #379

**Comment by aeosynth** [comment]
issue persists after #379:

```
# true && exit
thread 'main' panicked at 'called `Option::unwrap()` on a `None` value', /checkout/src/libcore/option.rs:335
note: Run with `RUST_BACKTRACE=1` for a backtrace.
ion: process ended by signal
```
